skfem/helpers.py

```diff
@@ -97,3 +97,63 @@ def eye(w, n):
     """Create diagonal matrix with w on diagonal."""
     return np.array([[w if i == j else 0. * w for i in range(n)]
                      for j in range(n)])
+
+
+def det(A):
+    """
+    Determinant of an array `A`
+    Parameters
+    ----------
+    A : (N, N,...) numpy.ndarray
+        Input array whose determinant is to be computed
+    Returns
+    -------
+    det : (...) numpy.ndarray
+        Determinant of `A`.
+    """
+    detA = zeros_like(A[0, 0])
+    detA = A[0, 0] * (A[1, 1] * A[2, 2] -
+                      A[1, 2] * A[2, 1]) -\
+        A[0, 1] * (A[1, 0] * A[2, 2] -
+                   A[1, 2] * A[2, 0]) +\
+        A[0, 2] * (A[1, 0] * A[2, 1] -
+                   A[1, 1] * A[2, 0])
+    return detA
+
+
+def inv(A):
```

Comment:
It should also be easy to make this work for 2x2 matrices.

Reply:
Ah I see, completely forgot that these were to be modified to include the `2x2` cases as well.